tests/hlsl-intrinsic/ray-tracing/rt-pipeline-intrinsics-ahit.slang

```diff
@@ -0,0 +1,155 @@
+//TEST:SIMPLE(filecheck=SPIRV):-target spirv-assembly -entry main -stage anyhit -profile glsl_460+GL_EXT_ray_tracing
+//   Motion rays not supported on HLSL impl currently, so disable with NOMOTION define:
+//TEST:SIMPLE(filecheck=DXIL):-target dxil-assembly -entry main -stage anyhit -profile sm_6_5 -DNOMOTION
+
+// SPIRV: OpEntryPoint
```

Comment:
Need to test at least the intrinsics/opcodes for all the intrinsic calls are there and are correct.